script.js

```diff
@@ -64,26 +54,24 @@ function makeImage(src, alt, className) {
 }
 
 function makeRecipeElement(recipeTitle, recipeDesc, recipeUrl) {
-
     const name = document.createElement('h3')
     name.textContent = recipeTitle
 
     const description = document.createElement('p')
     description.textContent = recipeDesc
 
-    const linkText = "Get the recipe"
     const link = document.createElement('a')
-    link.textContent = linkText
+    link.textContent = "Get the recipe"
```

Comment:
We don't really need to assign the "Get the recipie" text to a variable here, as it's only used once for `link.textContent`.